src/store.js

```diff
@@ -393,27 +393,37 @@ function makeLocalContext (store, namespace, path) {
   return local
 }
 
+let makeLocalGettersCache = {}
+let cacheStore = {}
+
 function makeLocalGetters (store, namespace) {
-  const gettersProxy = {}
-
-  const splitPos = namespace.length
-  Object.keys(store.getters).forEach(type => {
-    // skip if the target getter is not match this namespace
-    if (type.slice(0, splitPos) !== namespace) return
-
-    // extract local getter type
-    const localType = type.slice(splitPos)
-
-    // Add a port to the getters proxy.
-    // Define as getter property because
-    // we do not want to evaluate the getters in this time.
-    Object.defineProperty(gettersProxy, localType, {
-      get: () => store.getters[type],
-      enumerable: true
+  if (cacheStore !== store) {
```

Comment:
This way of clearing cache is not comprehensive. Getters may be changed via `registerModule`, `unregisterModule` or `hotUpdate`.

Reply:
Thank you so much for your reply! Could you give me a hint here?
I see that all those functions end up calling `resetStoreVM`, do you think that is the proper place to clear the cache? 
That way I could also get rid of that `cacheStore` variable